job-dsl-core/src/test/groovy/javaposse/jobdsl/dsl/helpers/publisher/PublisherContextSpec.groovy

```diff
@@ -5039,10 +5041,9 @@ class PublisherContextSpec extends Specification {
         value << [true, false]
     }
 
-    def 'svnTag with no options'() {
```

Comment:
why didn't codenarc tell me about the wrong indentation? Is there an codenarc check which could be enabled for the future?